lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -669,6 +669,70 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return this._buffer[ GETTER ]( idx*BYTES_PER_ELEMENT, this._isLE );
 	});
 
+	/**
+	* Returns a new typed array containing the elements in sorted order.
+	*
+	* @private
+	* @name toSorted
+	* @memberof TypedArray.prototype
+	* @type {Function}
+	* @param {Function} compareFcn - comparison function
+	* @throws {TypeError} `this` must be a typed array
+	* @throws {TypeError} first argument must be a function
+	* @returns {TypedArray} sorted array
+	*/
+	setReadOnly( TypedArray.prototype, 'toSorted', function sort( compareFcn ) {
+		var minIdx;
+		var minVal;
+		var obuf;
+		var buf;
+		var len;
+		var out;
+		var val;
+		var i;
+		var j;
+
+		if ( !isTypedArray( this ) ) {
+			throw new TypeError( format( 'invalid invocation. `this` is not %s %s.', CHAR2ARTICLE[ dtype[0] ], CTOR_NAME ) );
+		}
+		if ( arguments.length === 0 ) {
+			compareFcn = function defaultCompareFcn( a, b ) {
```

Comment:
We don't typically use function expressions, only declarations. I suggest refactoring accordingly. It is also not necessary to have this function in a nested scope and can be lifted to the top most scope.